pkg/instance/service.go

```diff
@@ -54,6 +64,107 @@ type userClientService interface {
 	FindGroupByName(token string, name string) (*models.Group, error)
 }
 
+func (s service) LinkDeploy(token string, oldInstance, newInstance *model.Instance) error {
+	err := s.Link(oldInstance.ID, newInstance.ID, newInstance.StackName)
+	if err != nil {
+		return err
+	}
+
+	oldStack, err := s.stackService.Find(oldInstance.StackName)
+	if err != nil {
+		return err
+	}
+
+	newStack, err := s.stackService.Find(newInstance.StackName)
+	if err != nil {
+		return err
+	}
+
+	// Consumed required parameters
+	for _, parameter := range newStack.RequiredParameters {
+		if parameter.Consumed && parameter.Name != newStack.HostnameVariable {
+			value, err := s.findParameterValue(parameter.Name, oldInstance, oldStack)
+			log.Println(parameter.Name, value)
```

Comment:
Could this print a plain text password? If so remove the line as we should not log such information.

I would even remove it regardless actually. To me such logs are debug logs that might only highlight that we need some tests.